echo-rest/src/test/groovy/com/netflix/spinnaker/echo/events/RestEventListenerSpec.groovy

```diff
@@ -26,7 +26,7 @@ import spock.lang.Subject
 class RestEventListenerSpec extends Specification {
 
   @Subject
-  RestEventListener listener = new RestEventListener()
+  RestEventListener listener = new RestEventListener(null, null, null)
```

Comment:
Add the `NoopRegistry()` as your last argument here

Reply:
cool, thanks